applications/RANSApplication/custom_elements/incompressible_potential_flow/incompressible_potential_flow_velocity_element.h

```diff
@@ -0,0 +1,294 @@
+//    |  /           |
```

Comment:
@KratosMultiphysics/potential-flow-team guys, considering that both the `RANSApplication` and the  `CompressiblePotentialFlowApplicataion` are expected to be merged to the `FluidDynamicsApplication` at some point, can you specify which are the differences of your element (aside of the Kutta condition) with respect to that one? Thanks!

As you are already deriving from the `FluidDynamicsApplication` we need to take this into account to avoid duplicating implementations.